src/components/SearchDropdown/index.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import Select, { ValueType } from 'react-select';
+
+export interface OptionsInterface {
+    label: string;
+    value: string;
+}
+
+interface OwnProps {
+    options: OptionsInterface[];
+    onSelect: (option: ValueType<OptionsInterface, false>) => void;
+    placeholder: string;
+    className?: string;
+}
+
+export const SearchDropdown: React.FC<OwnProps> = ({ className, onSelect, options, placeholder }) => {
+    const customStyles = {
```

Comment:
we are taking a different approach to styling the components in this repo. Please, use `pcss` files for styles instead of styles in tsx.